apps/native/app/src/ui/lib/accordion/accordion-item.tsx

```diff
@@ -33,19 +23,21 @@ const Icon = styled(Animated.View)`
   margin-right: ${({ theme }) => theme.spacing[1]}px;
 `
 
-const Chevron = styled(Animated.View)`
-  height: 25px;
-  width: 25px;
+const PlusMinus = styled(Animated.View)`
+  height: ${({ theme }) => theme.spacing[3]}px;
+  width: ${({ theme }) => theme.spacing[3]}px;
+  background-color: ${({ theme }) => theme.color.blue100};
+  border-radius: ${({ theme }) => theme.border.radius.circle};
+  justify-content: center;
+  align-items: center;
 `
 
-const Content = styled.View`
-  border-top-width: ${({ theme }) => theme.border.width.standard}px;
-  border-top-color: ${({ theme }) => theme.color.blue200};
-`
+const Content = styled.View``
 
 interface AccordionItemProps {
   children: React.ReactNode
   title: string
+  startOpen?: boolean
```

Comment:
I feel like startOpen is should just be `open?:  boolean` 🤔 . It is either opened or closed.

Reply:
I was trying to be more descriptive, since I already have a state variable called `isOpen`. I thought it would be confusing to have one `open` and another `isOpen`, but maybe that is okay 😄 Then the code would be:
```

   if (open && isOpen) {
      setOpen(false)
    }
    setIsOpen(!isOpen)
```

instead of:

```

   if (startOpen && isOpen) {
      setStartOpenOpen(false)
    }
    setIsOpen(!isOpen)
```

Neither of them is great 😂